src/components/Checkout/MonthlyGoodCTA.vue

```diff
@@ -186,6 +188,14 @@ export default {
 			]
 		};
 	},
+	computed: {
+		selectedAmountTracking() {
+			return ['MonthlyGood', 'click-amount-option', `[actual amount selected - ${this.mgOptionSelected}]`];
+		},
+		selectedCategoryTracking() {
+			return ['MonthlyGood', 'click-category-option', `[actual category select - ${this.selectedGroup}]`];
```

Comment:
Both of these would be easier to filter on and create reports from if the label was more concise. I think we can reduce these to `amount: ###` + `category: ###`.